pages/events.js

```diff
@@ -19,7 +19,27 @@ export async function getServerSideProps(context) {
   };
 }
 
+// const groupEvents = (events) => {
+//   const groups = {}
+//   events.forEach(event => {
+//     if (groups.hasOwnProperty(event.url)) {
+//       groups[event.url].usernames.push(event.username);
+//     } else {
+//       groups[event.url] = {
+//         ...event,
+//         usernames: [event.username]
+//       }
+//       delete groups[event.url].username
+//     }
+//   });
+//   return Object.values(groups);
+// }
+
+
 export default function Events({ events }) {
+
+  // const groupedEvents = groupEvents(events);
+
   let categorizedEvents = {
     all: events,
```

Comment:
since the `events` object is already grouped in the api call no need for the function